.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 npm-debug.log
 node_modules
+todo.md
```

Comment:
Oh damn. Sorry, that's my git noobishness. That's the file I've been keeping notes in. I removed that line.